### PR TITLE
Fix filter bug

### PR DESCRIPTION
**Pull request by alexbeeken**
fixes this bug

https://podio.com/hranswerlink-8ee92nawfl/issue-tracker/apps/product-feedback/items/2026

hrsc PR to use this fork instead of scryptmouse's

### CODE REVIEW

**Comment by thec0keman** [comment]
Are we at a place that we can get off of a fork and use their main app?

File: app/assets/javascripts/swagger-ui/lib/swagger-client.js [hunk 1]
```diff
@@ -1154,11 +1154,18 @@ Operation.prototype.getMissingParams = function(args) {
...
     if(typeof args[param.name] !== 'undefined') {
       if (param.in === 'body') {
-        body = args[param.name];
+        if(typeof body !== 'undefined') {
+          let json = JSON.parse(body);
+          json[param.name] = args[param.name];
+          body = JSON.stringify(json);
+        } else {
+          body = args[param.name];
+        }
```
Comment (thec0keman) [inline app/assets/javascripts/swagger-ui/lib/swagger-client.js:1168]: Is this enabling multiple entries in `body`?

**Review by dfockler — APPROVED** [review]

File: app/assets/javascripts/swagger-ui/lib/swagger-client.js (hunk 1 above)
Comment (alexbeeken) [reply app/assets/javascripts/swagger-ui/lib/swagger-client.js:1168]: Yes, and looking at the distributor docs this is exactly how our affiliate_id filter is working. 

```
In order to manage users for one of your other affiliates, simply include their affiliate_id in the root of your request payload. Make sure the affiliate_id is at the root, NOT embedded in the user payload, and that it is the correct affiliate ID for the user you wish to manage.
```

Comment (alexbeeken) [reply app/assets/javascripts/swagger-ui/lib/swagger-client.js:1168]: Before this, swagger was taking the affiliate_id and just replacing the user payload, which is why it was sending a malformed request body to the controller.

**Review by ekluff — APPROVED** [review]

**Pull request merged**